### PR TITLE
Add demo code for Appium automated testing

### PR DESCRIPTION
**Pull request by tkobialka**

### CODE REVIEW

File: conftest.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import os
+
+import pytest
+from appium import webdriver
+
+EXECUTOR = 'http://127.0.0.1:4723/wd/hub'
+ANDROID_APP_DIR = os.path.join(os.path.dirname(os.path.realpath(__file__)), 'android_apps')
+
+apk_files = [file for file in os.listdir(ANDROID_APP_DIR) if file.endswith('.apk')]
```
Comment (gtlambert) [inline conftest.py]: `file` is a Python keyword. Could we use `f` instead? Thanks!

File: LICENSE [hunk 2]
```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2019 Lambert Labs
```
Comment (gtlambert) [inline LICENSE]: `Lambert Labs Ltd` please

**Review by gtlambert — COMMENTED** [review]
Looks great! A couple of small things...

* could you add a short `README.md` explaining what the repository is for (ie a repo for a blog post) and link to the blog post?
* could you set up CircleCI with linting

Thanks Tom!

File: conftest.py (hunk 1 above)
Comment (tkobialka) [reply conftest.py]: Done

File: LICENSE (hunk 2 above)
Comment (tkobialka) [reply LICENSE]: Done

**Comment by tkobialka** [comment]
I had to downgrade urllib3 from 1.25.3 (installed locally by default) to 1.24.2 as it was causing conflicts:
```
urllib3=1.25.3
selenium=3.141.0 -> urllib3
conda[version='>=4.7.12'] -> requests[version='>=2.18.4,<3'] -> urllib3[version='>=1.21.1,<1.23|>=1.21.1,<1.24|>=1.21.1,<1.25']
```
It looks like the conflict originates from the conda version itself. The conda package does not appear in the package list of my local environment because it is not the base environment. On the other hand CircleCI works off the base environment, which does have the conda package. That looks like the reason why I wasn't getting this conflict before...

I also upgraded Python to 3.7.4 to match that in the latest conda version.

**Review by gtlambert — APPROVED** [review]

**Pull request merged**